src/writeFile.js

```diff
@@ -1,7 +1,11 @@
 // const filePath = require('../files')
-const fs = require('fs')
-const PATH = require('path')
-const srcUtils = require('./../src/utils')
+
+// @TODO instead of importing whole object here - which will increase a size of a bundle, deconstruct it and import only methods that we're using here
```

Comment:
@svr8 can you solve this 3 todos that i just added? Vadim will be able to review your PR only at the morning tomorrow, so we have some time before this PR will be approved/merged

Reply:
Ok, I have updated following the TODOs. I made a slight change in imports...
This file is using a function `writeFile` from `fs` module and declares a new function `writeFile`. I have imported the `writeFile` in `fs` as `fsWriteFile` and renamed wherever needed [here](https://github.com/GroceriStar/food-static-files-generator/blob/05093448a2474d0e8e3ff9f84ab63da1228cb81b/src/writeFile.js#L3).

Reply:
you mentioned a good point here. FS has `writeFile` so we have the same name for our method. at it can be a big problem = source of confusion.
Can you rename our method to `write`(i didn't have any better name) and recursively change it at all places(skip projects folder) - at our 6-8 files that we have?

Reply:
I have just updated the changes

Reply:
perfect. resolving